tests/pass-dep/libc/libc-socketpair.rs

```diff
@@ -1,4 +1,5 @@
 //@ignore-target-windows: No libc socketpair on Windows
+//@compile-flags: -Zmiri-preemption-rate=0
```

Comment:
Why would this test disable preemption...?

Reply:
Oh, it contains the race test as well.
```suggestion
// test_race depends on a deterministic schedule.
//@compile-flags: -Zmiri-preemption-rate=0
```